firefox-ios/Client/Experiments/RecordedNimbusContext.swift

```diff
@@ -126,6 +126,7 @@ class RecordedNimbusContext: RecordedContext {
             daysOpenedInLast28: eventQueryValues[RecordedNimbusContext.DAYS_OPENED_IN_LAST_28].toInt64()
         )
 
+        GleanMetrics.Pings.shared.nimbus.setEnabled(enabled: true)
```

Comment:
I wondered why this is unconditionally enabled here.
Am I correct in that this `record` code is never called if Nimbus is not enabled (= the user disabled "Install and run studies")?

Reply:
Hmm this is a good point.. lemme double check

Reply:
Asked differently: under which circumstances would telemetry be disabled, but Nimbus enabled?

If that is not possible, then I think you shouldn't use `follows_collection_enabled` for this ping. It can just follow the regular Glean preference.

Reply:
Ohhhhhh I understand now. Yeah that's a good callout, I'll remove that field from the `pings.yaml`